plugins/binaryninja/binjastub/__init__.py

```diff
@@ -1,11 +1,21 @@
 import os
+import sys
+from distutils.dir_util import copy_tree
 from pathlib import Path
 
 import z3
 
+DIR = os.path.dirname(__file__)
+
 # Hack to include z3 path for virtualenv installs
 bin_path = Path(z3.__file__).parent.parent / "bin"
 if bin_path.exists():
     os.environ["PATH"] = str(bin_path) + ":" + os.environ.get("PATH", "")
 
+# Add parent directory to sys.path
+sys.path.append(os.path.join(DIR, ".."))
```

Comment:
What was broken before? Unable to import the plugin?

Reply:
Yep, because the directory that binary ninja runs from changed after we reorganised the directory structure.